tests/integration/tests/extensibility/ext-test-variables.spec.js

```diff
@@ -0,0 +1,239 @@
+      },
+    );
+
+    cy.contains('Submit').click();
+
+    cy.get('.fd-dialog__body')
+      .find('.sap-icon--message-success')
+      .should('have.length', 2);
+  });
+
+  it('Go to variables', () => {
+    cy.loginAndSelectCluster({
+      fileName: 'kubeconfig-k3s.yaml',
+      storage: 'Session storage',
+    });
+    cy.contains('Namespaces').click();
+
+    cy.contains('a', NAMESPACE).click();
+
+    cy.getLeftNav()
+      .contains('Testin')
+      .click();
+
+    cy.getLeftNav()
+      .contains('Test Resources')
+      .click();
+
+    cy.contains('Create Test Resource').click();
+
+    cy.get('.fd-dialog__content').as('form');
+  });
+
+  it('Tests variables', () => {
+    cy.get('.fd-dialog__content').as('form');
+
+    // test vars with no default value
+    cy.get('@form')
+      .find('[data-testid="Simple / Advanced"]:visible')
+      .find('input')
+      .should('be.empty');
+
+    // test vars with enums
+    cy.get('@form')
+      .find('[data-testid="Simple / Advanced"]:visible')
+      .find('span')
+      .find('[aria-label="Combobox input arrow"]:visible', { log: false })
+      .click();
+
+    cy.get('[role="list"]')
+      .contains('simple')
+      .should('exist');
+
+    cy.get('[role="list"]')
+      .contains('advanced')
+      .should('exist');
+
+    // test if fielsd based on visibility are not visible
+    cy.get('@form')
+      .find('[data-testid="spec.name"]:visible')
+      .should('not.exist');
+
+    cy.get('[aria-label="expand Advanced"]:visible', { log: false }).should(
+      'not.exist',
+    );
+
+    // test visibility based on var (select 'simple')
+    cy.get('[role="list"]')
+      .contains('simple')
+      .click();
+
+    cy.get('@form')
+      .find('[data-testid="spec.name"]:visible')
+      .should('exist');
+
+    // test visibility based on var (select 'advanced')
+    cy.get('@form')
+      .find('[data-testid="Simple / Advanced"]:visible')
+      .find('span')
+      .find('[aria-label="Combobox input arrow"]:visible', { log: false })
+      .click();
+
+    cy.get('[role="list"]')
+      .contains('advanced')
+      .click();
+
+    cy.get('[aria-label="expand Advanced"]:visible', { log: false }).should(
+      'exist',
+    );
+
+    // test vars with defaultValue
+    cy.get('@form')
+      .find('[data-testid="varWithDefaultValue"]:visible')
+      .should('have.value', 'default');
+
+    // test vars with dynamicValue
+    cy.get('@form')
+      .find('[data-testid="varWithDynamicValue"]:visible')
+      .should('have.value', 'dynamic name');
+  });
+
+  it('Tests presets', () => {
+    cy.get('.fd-dialog__content').as('form');
+    // test default preset
+    cy.get('@form')
+      .find('[arialabel="TestResource name"]:visible')
+      .should('have.value', NAME);
+
+    // test presets
+    cy.get('@form')
+      .find('.fd-select__text-content:visible')
+      .contains('Choose preset')
+      .next('.fd-button.fd-select__button')
+      .click();
```

Comment:
```suggestion
    cy.get('@form')
      .contains('Choose preset')
      .click();
```

I think it's enough to find the preset dropdown and click it.